### PR TITLE
Add workaround for wrongly named aarch64 compiler

### PR DESCRIPTION
**Pull request by Tombana**

### CODE REVIEW

File: docs/compute-engine/build_arm.md [hunk 1]
```diff
@@ -96,6 +96,23 @@ example program `lce_minimal` and benchmark program `benchmark_model`.
...
 `libtensorflow-lite.a` which includes the LCE customs ops.
 
+
+Note: on some systems the compiler is incorrectly named `aarch64-unknown-linux-gnu-gcc`
+while it should be named `aarch64-linux-gnu-gcc`. If building with the option
+`--aarch64` results in errors then the following bash script can be used to
+create symlinks that fix this naming issue.
+
+```bash
+#!/bin/bash
```
Comment (lgeiger) [inline docs/compute-engine/build_arm.md]:
```suggestion
#!/usr/bin/env bash
```

File: docs/compute-engine/build_arm.md [hunk 2]
```diff
@@ -96,6 +96,23 @@ example program `lce_minimal` and benchmark program `benchmark_model`.
 In the `lib` folder, you can find the TensorFlow Lite static library
 `libtensorflow-lite.a` which includes the LCE customs ops.
 
+
+Note: on some systems the compiler is incorrectly named `aarch64-unknown-linux-gnu-gcc`
```
Comment (lgeiger) [inline docs/compute-engine/build_arm.md]: Side note: If you want to make it stick out on the website, you can do something [like this](https://squidfunk.github.io/mkdocs-material/extensions/admonition/#removing-the-title).

Comment (Tombana) [reply docs/compute-engine/build_arm.md]: That looks nice. I updated it, where can i preview if it worked?

Comment (lgeiger) [reply docs/compute-engine/build_arm.md]: The view deployment button should show it:
Screenshot 2020-02-26 at 15 56 47

https://docs-ixzqp6vnn.now.sh/compute-engine/build_arm/

File: docs/compute-engine/build_arm.md [hunk 3]
```diff
@@ -96,6 +96,24 @@ example program `lce_minimal` and benchmark program `benchmark_model`.
 In the `lib` folder, you can find the TensorFlow Lite static library
 `libtensorflow-lite.a` which includes the LCE customs ops.
 
+
+!!! note ""
+    Note: on some systems the compiler is incorrectly named `aarch64-unknown-linux-gnu-gcc`
```
Comment (lgeiger) [inline docs/compute-engine/build_arm.md]:
```suggestion
    On some systems the compiler is incorrectly named `aarch64-unknown-linux-gnu-gcc`
```

**Comment by lgeiger** [comment]
Looks good https://docs-pekfg9bk3.now.sh/compute-engine/build_arm/

**Comment by AdamHillier** [comment]
Ah yes thank you for this, this is the problem I faced earlier @arashb

**Comment by arashb** [comment]
Yes, that’s why i told @tom to update the doc 😉

**Pull request merged**